lib/jekyll/converters/scss.rb

```diff
@@ -72,11 +72,16 @@ def sass_dir_relative_to_site_source
       end
 
       def sass_load_paths
+        paths = (user_sass_load_paths + [sass_dir_relative_to_site_source])
+
         if safe?
-          [sass_dir_relative_to_site_source]
-        else
-          (user_sass_load_paths + [sass_dir_relative_to_site_source]).uniq
-        end.select { |load_path| File.directory?(load_path) }
+          paths.map! { |path| Jekyll.sanitized_path(@config["source"], path) }
+        end
+
+        # Expand file globs (e.g. `node_modules/*/node_modules` )
+        paths = paths.map { |path| Dir.glob(path) }.flatten.uniq
+
+        paths.select { |path| File.directory?(path) }
```

Comment:
Should this line and line 82 be combined into one string rather than re-assigning `paths`? Thus `paths.map { }.flatten.uniq.select { }`.